recipes/upsetplot/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "UpSetPlot" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0ad40e6c5bc614bb38dff10d1c4a12ab3e20ea2f978ada8d67ba5ad3bbf195f2
+
+build:
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+    - pandas
+    - matplotlib >=2.0
+  run:
+    - python
+    - pandas
+    - matplotlib >=2.0
+
+test:
+  # Some packages might need a `test/commands` key to check CLI.
+  # List all the packages/modules that `run_test.py` imports.
+  imports:
+    - upsetplot
+
+about:
+  home: https://github.com/jnothman/UpSetPlot
+  license: BSD
```

Comment:
Which version of the BDS license?

Reply:
I've specified now "New BSD License". Is that okay as such?

Reply:
Seems it cannot include the word "License" according to the linter. Let's try just "New BSD" then.

Reply:
Yeah, "BSD-3-Clause" is usually used.

Reply:
Alright, will change.